crashtracker/Cargo.toml

```diff
@@ -12,7 +12,7 @@ crate-type = ["lib"]
 
 [target.'cfg(unix)'.dependencies]
 # TODO: move to a named release once https://github.com/libbpf/blazesym/pull/520 makes it to release
-blazesym = {git = "https://github.com/libbpf/blazesym.git", rev = "4fa15cb"}
+blazesym = {git = "https://github.com/libbpf/blazesym.git", rev = "v0.2.0-alpha.11"}
```

Comment:
No need to specify the git repo, since this is published to crates.io

Reply:
Although this is true there might by value in keeping a consistent setting (in terms of version), and the C ffi parts are not systematically published.